Initial release of pl-PL

Daxxxis opened the pull request:
The translation document is not available, so I create a PR right away. This is a full translation into Polish (pl-PL)

simon-knuth commented:
Hi Daxxxis,

thank you so much for your contribution, I merged the PR. 
Unfortunately, this exact translation probably won't make it to a Microsoft Store release. I am still working on rewriting the app, including the UI and many of its strings. The experience will change a lot, so sticking to the same old set of strings is not going to be possible.

I hope this isn't a huge disappointment. I'd love to see you update this translation to take the changes/additions into account, once my work on v2.0 is finished.

Daxxxis commented:
Hello. Of course, as soon as version 2.0 is available, I will update the translation accordingly. This is not a problem for me. I would only ask for notification that version 2 is ready for translation.
Thanks.

simon-knuth commented:
Hey there,

it’s taken some time, but this month I’m finally ready to tie up a few loose ends before updating the app. I updated the [page](https://simon-knuth.github.io/scanner/help-translate.html) with fancy POEditor projects and also included the store strings this time around. If you‘re still in the mood for this task, I‘d love to see a Polish translation included with the new release. Stay safe ✌️

Daxxxis commented:
Hi. Of course, I am already getting to it. Thank you for this great app and for the work you put into it 😃

Daxxxis commented:
@simon-knuth 
The translation is finished. If it is possible, I would like to check what it looks like in the app (to eliminate errors), if it is possible, of course.

simon-knuth commented:
@Daxxxis
Damn you're quick, thank you 😮 Polish seems to produce even longer sentences than German, so I already optimized some word wrapping based on your translation. I also noticed an inconsistency in the English translation of `TextSettingsFeedback3.Text` on POEditor, so I believe you'll need to take a look at that one - sorry. There might also be a space missing in `TextSettingsTranslations1.Text`, but that's just a guess.

I completely forgot that translating strings without ever having seen the app must be pretty inconvenient, so that's definitely a good idea: You can check out [this package](https://github.com/simon-knuth/scanner/raw/temp/Scanner%20v2%20test%20package.zip), which includes your exported translations. You _should_ be able to install the app by extracting the folder and then running `Install.ps1` with PowerShell (right-click the file and select "Run with PowerShell"). I haven't tested the compatibility yet, but the app at least runs on Windows 10 version 20H2. You can also test some contextual dialogs using the debug menu (hold down the CTRL key while navigating to settings, then scroll to the bottom).

Daxxxis commented:
@simon-knuth 
I have just fixed some glaring translation errors that I noticed after reviewing the application. I also corrected what you wrote about, thanks for vigilance, there should indeed be a space 😄.
I also noticed some confusing app behavior. If you enable page selection, you can close the page list and the editing tools will not be available. It looks like an unintended "feature". I am enclosing a screenshot:
![image](https://user-images.githubusercontent.com/60961958/110406609-ce4a1f00-8082-11eb-858f-65366979ee8c.png)

simon-knuth commented:
@Daxxxis 
Those are some great points, I completely forgot that the Display Name of the installed app is separate from the Short Name in the Store, I'll definitely need to take a look at the app's manifest localization...

I originally postponed this issue with the selection mode as low-priority, but I agree this could be really confusing especially since the page management can be hidden on small/medium-sized displays. It'll probably require some work, but I'll figure out some solution for that prior to the release. Thank you, I still needed to finish the event logging before I would have let some people test the app, so this is the first feedback I'm getting since I started working on the update :)

One more thing: Is the panel with the scan options never showing up as acrylic for you or is this just a side-effect of capturing the screenshot? It should look something like this:

![image](https://user-images.githubusercontent.com/50021001/110551386-ad8ed180-8135-11eb-9699-791a3274796a.png)

Daxxxis commented:
![image](https://user-images.githubusercontent.com/60961958/110552422-99e46a80-8137-11eb-9fe3-192d9038da95.png)
It was either by taking a screenshot or temporarily disabled transparency effects for no reason by Windows Insider build.
